modules/release-notes/ref-release-notes-new-features.adoc

```diff
@@ -4,5 +4,65 @@
+
+The available counters include the following:
+
+`backend_keycloak_fetch_task_failure_count_total`: Counts fetch task failures where no data was returned due to an error.​
+
+`backend_keycloak_fetch_data_batch_failure_count_total`: Counts partial data batch failures. Even if some batches fail, the plugin continues fetching others.​
+
+These counters include the `taskInstanceId` label, which uniquely identifies each scheduled fetch task, and allows you to trace failures back to individual task executions.
+
+Example configuration:
+
+```text
+backend_keycloak_fetch_data_batch_failure_count_total{taskInstanceId=&#34;df040f82-2e80-44bd-83b0-06a984ca05ba&#34;} 1
+```
+
+You can export metrics using any OpenTelemetry-compatible backend, such as **Prometheus**.
+
+
+[id="enhancement-rhidp-5987"]
+== Updated Auditor Service
+
+{product} {product-version} introduces an enhancement to the RBAC and Bulk Import plugins, enabling users to utilize Backstage&#39;s new Auditor service. The key features include: ​
+
+* Audit log format update: 
+
+The audit log format has been updated to align with the new Auditor service conventions. Audit fields and event identifiers have been updated. Filtering queries based on the old format may no longer function as expected.​
+
+* Backend Plugin API integration: 
+
+The audit log is now backed by the `@backstage/backend-plugin-api` package.​
+
+* Audit events Grouping: 
+
+The Bulk Import backend plugin emits audit events for various operations, with the events are grouped logically by `eventId`.​
+
+
+
+
+[id="feature-rhidp-6253"]
+== {product-short} community plugins updated to Backstage 1.36
+
+The {product-short} community plugins have been updated to Backstage version 1.36.
+
+[id="feature-rhidp-6269"]
+== Added a new RBAC conditional rule `IS_OWNER` to RBAC plugin
+
+{product} introduces a new RBAC conditional rule, `IS_OWNER`, that allows administrators to assign ownership to roles and control access to the RBAC plugin. This enhancement enables more granular access control by allowing ownership-based filtering of roles, permission policies, and conditional policies.
+
+This enhancement removes the resource type from the `policy.entity.create` permission, preventing conditional rules from being applied to the permission. You can update all permission policies that utilize the resource type `policy-entity` with the action `create` (for example `role:default/some_role, policy-entity, create, allow` to `role:default/some_role, policy.entity.create, create, allow`) to prevent degradation in the future.
+
+
+[id="enhancement-rhidp-6882"]
+== Default OIDC sign-in resolver updated
+
+With this update, the default resolver for OIDC sign-in is set to `oidcSubClaimMatchingKeycloakUserId` to enhance security. This resolver is now also available as a configurable option under the sign-in resolver settings.
```

Comment:
I believe the resolver is `oidcSubClaimMatchingIdPUserId` and I think this might be a breaking change but I'll let @JessicaJHee confirm

Reply:
The user-facing sign-in resolver to configure is `oidcSubClaimMatchingKeycloakUserId`, so it's correct in this case. The `oidcSubClaimMatchingIdPUserId` resolver is used internally to support multiple OIDC providers and is not intended for direct configuration.  

I think enhancement is good for this as it doesn't impact current behaviour/configs.